catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -274,8 +264,6 @@
                             quartz-jobs,
                             c3p0,
                             terracotta-toolkit-runtime-ee,
```

Comment:
Sorry for the delayed response. I tried to remove that dependency as well. It didn't appear in the manifest to my knowledge but one of the itests fail. I have to sanity check this PR again tonight. I will test removing that as well, I can run parallel builds.